tests/google.js

```diff
@@ -5,33 +5,34 @@ const sleep = 3000;
 
 async function runTest() {
   // abre o navegador
-  const driver = await new Builder().forBrowser(Browser.CHROME).build();
+  const driver = await new Builder().forBrowser(Browser.CHROME).build()
 
   // maximiza a janela do browser
-  await driver.manage().window().maximize();
+  await driver.manage().window().maximize()
 
-  await driver.sleep(sleep);
+  await driver.sleep(sleep)
 
   // acessa o google
-  await driver.get('https://google.com');
+  await driver.get("https://google.com");
 
   await driver.sleep(sleep);
 
   // pesquisa por 'programming memes'
-  await driver.findElement(By.name('q')).sendKeys('programming memes', Key.RETURN);
+  await driver.findElement(By.name('q')).sendKeys("programming memes", Key.RETURN);
 
-  await driver.sleep(sleep);
+        await driver.sleep(sleep);
 
   // localiza o botão de "imagens"
   const imagesTab = await driver.findElement(By.xpath('//*[@id="hdtb-msb"]/div[1]/div/div[2]/a'));
 
   // clica no botão
   await imagesTab.click();
 
-  await driver.sleep(sleep);
+
+      await driver.sleep(sleep);
 
   // fecha o navegador
-  await driver.close();
+    await driver.close();
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/mauricio-cantu/selenium-presentation/pullRequest?prid=9606874)
```suggestion
  await driver.close();
```